### DISCUSSION TITLE
cant start server no matter what type i try.

### DISCUSSION THREAD

**Discussion by BamsyTheSergal** [itzg/docker-minecraft-server#1924] [General]
https://i.imgur.com/fIfPFrl.png
log(?)
this just repeats over and over if i use forge, and just gets stuck at the first 3 lines if i do anything else.

https://i.imgur.com/S1QqOjc.png
docker compose file

**Comment by BamsyTheSergal** [comment]
i switch linux distros and it worked.

**Reply by itzg** [reply]
Glad to hear and thanks for following up.
